cirq/ops/sycamore_gate.py

```diff
@@ -0,0 +1,40 @@
+"""Gates for Google's Sycamore architecture."""
+
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+from cirq import ops, protocols
+
+if TYPE_CHECKING:
+    import cirq
+
+
+class SycamoreGate(ops.FSimGate):
+    """Sycamore two-qubit gate is a FSimGate(π/2, π/6) gate.
+
+    The unitary of this matrix is
+
+        [[1, 0, 0, 0],
+         [0, 0, -1j, 0],
+         [0, -1j, 0, 0],
+         [0, 0, 0, exp(- 1j * π/6)]]
+    """
```

Comment:
Comment that this gate is a gate that is "easy" to get on the Sycamore chip and add a link to the nature publication